torch/nn/modules/batchnorm.py

```diff
@@ -778,8 +787,7 @@ def forward(self, input: Tensor) -> Tensor:
 
     @classmethod
     def convert_sync_batchnorm(cls, module, process_group=None):
-        r"""Helper function to convert all :attr:`BatchNorm*D` layers in the model to
-        :class:`torch.nn.SyncBatchNorm` layers.
+        r"""Convert all :attr:`BatchNorm*D` layers in the model to :class:`torch.nn.SyncBatchNorm` layers.
```

Comment:
The initial phrasing looks ok to me, did pydocstyle complain here?

nit:
```suggestion
        r"""Converts all :attr:`BatchNorm*D` layers in the model to :class:`torch.nn.SyncBatchNorm` layers.
```

Reply:
Yes, the issue is pydocstyle D401 error code "First line should be in imperative mood". `Helper` does not conform with this.